coffee-chats/src/main/java/com/google/step/coffee/entity/ChatRequest.java

```diff
@@ -0,0 +1,54 @@
+package com.google.step.coffee.entity;
+
+import java.util.Date;
+import java.util.List;
+
+public class ChatRequest {
+
+  private List<String> tags;
+  private List<Date> dates;
+  private int minPeople;
+  private int maxPeople;
+  private int duration;
+  private boolean matchRandom;
+  private boolean matchRecents;
+
+  public ChatRequest(List<String> tags, List<Date> dates, int minPeople, int maxPeople,
```

Comment:
Nit: This constructor takes many arguments, do you think some could be abstracted away?

Reply:
Would this be a good match for a builder pattern? Or even, protobuf?

Reply:
I created a builder to make a more fluent interface to create these objects instead of calling the constructor directly. What specifically do you mean by abstracted away?